src/views/virtualmachines/list/utils/totalsQueries.ts

```diff
@@ -0,0 +1,29 @@
+import { ALL_NAMESPACES_SESSION_KEY } from '@kubevirt-utils/hooks/constants';
+
+export const VMTotalsQueries = {
+  TOTAL_CPU_REQUESTED: 'TOTAL_CPU_REQUESTED',
+  TOTAL_CPU_USAGE: 'TOTAL_CPU_USAGE',
+  TOTAL_MEMORY_USAGE: 'TOTAL_MEMORY_USAGE',
+  TOTAL_STORAGE_CAPACITY: 'TOTAL_STORAGE_CAPACITY',
+  TOTAL_STORAGE_USAGE: 'TOTAL_STORAGE_USAGE',
+};
+
+export const getVMTotalsQueries = (namespace: string, namespacesList: string[]) => {
+  let byNamespace = ' by (namespace)';
+  let namespaceFilter = `{namespace='${namespace}'}`;
+  let namespaceFilterCpu = `namespace='${namespace}'`;
+
+  if (namespace === ALL_NAMESPACES_SESSION_KEY) {
+    byNamespace = '';
+    namespaceFilter = '';
+    namespaceFilterCpu = `namespace=~'${namespacesList.join('|')}'`;
+  }
```

Comment:
we don't like to use let. 

Can you do something like 

```suggestion
  const byNamespace = namespace !== ALL_NAMESPACES_SESSION_KEY ? ' by (namespace)' : '';
  const namespaceFilter = namespace !== ALL_NAMESPACES_SESSION_KEY ?`{namespace='${namespace}'}` : '';
  const namespaceFilterCpu =namespace !== ALL_NAMESPACES_SESSION_KEY ? `namespace='${namespace}'` :`namespace=~'${namespacesList.join('|')}'` ;
```

What do you think?

Reply:
yeah, I usually don't like it either but here it seemed cleaner to me than using the same ternary 3 times